gfreshii-webapp/src/main/java/com/edbootcamp/view/RecipeViewImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.edbootcamp.view;
+
+import java.util.List;
+
+import com.edbootcamp.api.views.RecipeView;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+//TODO Delete all the Hibernates stuff,  add JSON annotations as needed for the UI
+
+public class RecipeViewImpl implements RecipeView {
```

Comment:
RecipeView and RecipeImpl should implement Recipe ( I )

Reply:
renamed